redux/apis/auth-api.ts

```diff
@@ -49,19 +50,28 @@ export const authApi = createApi({
       }),
     }),
 
-    verifyLink: builder.query<VerifyEmailResponse | any, string>({
+    verifyLink: builder.query<AuthenticationResponse | any, string>({
```

Comment:
change the related api name or the interface name in order to match so that it's easier to track back later

Reply:
@imrankhan-stixor , same here